<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QDialog" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="exitPush">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addModuleButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>145</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить Модуль</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>471</width>
     <height>271</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>469</width>
      <height>269</height>
     </rect>
    </property>
    <widget class="QLabel" name="emptyLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>261</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>              Нет добавленных модулей</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="favoriteButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>145</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>Избранное</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
